controllers/drplacementcontrol.go

```diff
@@ -162,6 +164,9 @@ func (d *DRPCInstance) RunInitialDeployment() (bool, error) {
 
 	d.setProgression(rmn.ProgressionCompleted)
 
+	rmnutil.ReportIfNotPresent(d.reconciler.eventRecorder, d.instance, corev1.EventTypeNormal,
```

Comment:
This event is reported by setDRState within `startDeploying` or if empty just above in line 157. This would be a repeat event and also not required.